neptunecontrib/monitoring/pytorch_lightning.py

```diff
@@ -29,27 +29,28 @@
 
 class NeptuneLogger(LightningLoggerBase):
     r"""
-    Log using `Neptune <https://neptune.ai>`_.
+    .. warning::
+        This is LegacyLogger implementation that is under maintenance support only.
+        No new updates will be made to this logger.
 
-    Install it with pip:
+        Please visit integration
+        `docs <https://docs.neptune.ai/integrations-and-supported-tools/model-training/pytorch-lightning>`_
+        to learn about the latest, fully supported version.
 
-    .. code-block:: bash
-
-        pip install neptune-client
-
-    The Neptune logger can be used in the online mode or offline (silent) mode.
+    The legacy Logger can be used in the online mode or offline (silent) mode.
```

Comment:
```suggestion
    The legacy Logger can be used in online mode or silent mode.
```
I would avoid using "offline" not for the new offline mode unless we cannot avoid it (like param name :D).